ios/FluentUI/ActivityIndicator/ActivityIndicator.swift

```diff
@@ -143,8 +145,10 @@ class MSFActivityIndicatorStateImpl: NSObject, ObservableObject, ControlConfigur
 
     init(size: MSFActivityIndicatorSize) {
         self.size = size
-        self.tokens = ActivityIndicatorTokens(size: size)
 
         super.init()
+
+        // Ensure `tokens` has a weak reference back to this object to fetch `size` and `style`.
+        self.tokens.state = self
```

Comment:
Now that we're using unowned instead of weak, we should probably update the comments as well

Reply:
Will do! I'll also make sure all the comments reference the correct variables.